packages/desktop-client/src/components/accounts/AccountSyncCheck.jsx

```diff
@@ -17,31 +20,37 @@ function getErrorMessage(type, code) {
     case 'ITEM_ERROR':
       switch (code.toUpperCase()) {
         case 'NO_ACCOUNTS':
-          return 'No open accounts could be found. Did you close the account? If so, unlink the account.';
+          return t(
+            'No open accounts could be found. Did you close the account? If so, unlink the account.',
+          );
         case 'ITEM_LOGIN_REQUIRED':
-          return 'Your password or something else has changed with your bank and you need to login again.';
+          return t(
+            'Your password or something else has changed with your bank and you need to login again.',
+          );
         default:
       }
       break;
 
     case 'INVALID_INPUT':
       switch (code.toUpperCase()) {
         case 'INVALID_ACCESS_TOKEN':
-          return 'Item is no longer authorized. You need to login again.';
+          return t('Item is no longer authorized. You need to login again.');
         default:
       }
       break;
 
     case 'RATE_LIMIT_EXCEEDED':
-      return 'Rate limit exceeded for this item. Please try again later.';
+      return t('Rate limit exceeded for this item. Please try again later.');
 
     case 'INVALID_ACCESS_TOKEN':
-      return 'Your SimpleFIN Access Token is no longer valid. Please reset and generate a new token.';
+      return t(
+        'Your SimpleFIN Access Token is no longer valid. Please reset and generate a new token.',
+      );
 
     case 'ACCOUNT_NEEDS_ATTENTION':
       return (
         <>
-          The account needs your attention at{' '}
+          <Trans>The account needs your attention at</Trans>{' '}
```

Comment:
This translation unit should also include the `<Link>` below to form a complete sentence.